### PR TITLE
Allow all Fixnum statuses

### PR DESCRIPTION
**Pull request by dabrorius**
This is a fixup for #952
I didn't edit changelog because #952 change description still seems valid.

### CODE REVIEW

**Comment by dblock** [comment]
@dm1try Double-check this for me please? Merge at will.

**Comment by dblock** [comment]
@dabrorius I think we need an explicit test for being able to set a status to a value not in that Rack list.

File: spec/grape/dsl/inside_route_spec.rb [hunk 1]
```diff
@@ -118,11 +118,6 @@ def initialize
             .to raise_error(ArgumentError, 'Status code :foo_bar is invalid.')
         end
 
-        it 'raises error if unknow status code is passed' do
-          expect { subject.status 210 }
-            .to raise_error(ArgumentError, 'Status code 210 is invalid.')
```
Comment (dblock) [inline spec/grape/dsl/inside_route_spec.rb]: So this test should be here, but assert the opposite, `.to_not raise_error` ...

Comment (dabrorius) [inline spec/grape/dsl/inside_route_spec.rb]: Ok, I thought we will just pretend like that feature never existed :)

File: spec/grape/dsl/inside_route_spec.rb [hunk 2]
```diff
@@ -118,9 +118,8 @@ def initialize
...
+        it 'accepts unknown Fixnum status codes' do
+          expect { subject.status 210 }.to_not raise_error
```
Comment (dblock) [inline spec/grape/dsl/inside_route_spec.rb]: Can I be annoying? I think we also need an integration level test that says an endpoint actually returns this status. Thanks.

Comment (dabrorius) [inline spec/grape/dsl/inside_route_spec.rb]: Sure, I'll see what I can do.

**Comment by dabrorius** [comment]
@dblock I added few integration specs. Let me know if that's what you had in mind.

**Comment by dblock** [comment]
Good. Will merge on :green_book:

**Pull request merged**